lib/sequent/configuration.rb

```diff
@@ -109,6 +111,7 @@ def initialize
       self.offline_replay_persistor_class = DEFAULT_OFFLINE_REPLAY_PERSISTOR_CLASS
       self.online_replay_persistor_class = DEFAULT_ONLINE_REPLAY_PERSISTOR_CLASS
       self.database_config_directory = DEFAULT_DATABASE_CONFIG_DIRECTORY
+      self.database_schema_directory = DEFAULT_DATABASE_SCHEMA_DIRECTORY
```

Comment:
My reading of the code above (line 15) is that `db` is the current default for new installs, so I set it to that so that the behaviour doesn't change. 

If an existing site has `database_config_directory` configured as '/config`, then as you say it would be best to use that value so that it doesn't break. However, this would stop the new site generator from working. Would it make more sense to alter the install generator too? I think that would need a load of other updates to the docs. Happy to do that too if you feel it fits in this PR.

Alternatively, how about this?

```
self.database_schema_directory = self.database_config_directory
```

I think that would fix both cases for now.

Reply:
I meant this in the context of providing the deprecation warning. So yes in the first new minor version we do:

`self.database_schema_directory = self.database_config_directory` and provide a deprecation warning that the default location will change.

After that we change it to:

`self.database_schema_directory = DEFAULT_DATABASE_SCHEMA_DIRECTORY `